lib/ui/page/home/page/user/view.dart

```diff
@@ -364,15 +314,40 @@ class UserView extends StatelessWidget {
         //     contact ? SvgIcons.deleteContact16 : SvgIcons.addContact16,
         //   ),
         // ),
-        // ActionButton(
-        //   text: favorite
-        //       ? 'btn_delete_from_favorites'.l10n
-        //       : 'btn_add_to_favorites'.l10n,
-        //   onPressed: favorite ? c.unfavoriteContact : c.favoriteContact,
-        //   trailing: SvgIcon(
-        //     favorite ? SvgIcons.favorite16 : SvgIcons.unfavorite16,
-        //   ),
-        // ),
+        ActionButton(
```

Comment:
Aren't the icons a bit off? I can tell just by looking at the existing ones (without comparing those to the screenshot) that `delete` icon is totally too small. Don't you think?

<img width="962" alt="Screenshot 2025-06-18 at 13 05 08" src="https://github.com/user-attachments/assets/ff8d927c-8371-4d80-b290-206313efe9c1" />